pkg/cmd/cli/cmd/create.go

```diff
@@ -54,8 +52,8 @@ func CmdCreate(f *clientcmd.Factory, reader io.Reader, out io.Writer) *cobra.Com
 		},
 	}
 
-	cmd.Flags().Int("masters", 1, "Numbers of workers in spark cluster")
-	cmd.Flags().Int("workers", 1, "Numbers of workers in spark cluster")
+	cmd.Flags().Int("masters", 0, "Numbers of workers in spark cluster")
+	cmd.Flags().Int("workers", 0, "Numbers of workers in spark cluster")
 	cmd.Flags().String("masterconfigdir", defaultsparkconfdir, "Config folder for spark master")
```

Comment:
Yes, I'll take those out.  SPARK_CONF_DIR should be left alone and set by oshinko-core to a fixed path iff a master or worker config is named.